src/Elastic.Apm/Logging/IApmLoggingExtensions.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Net.Http;
+using System.Runtime.InteropServices;
+
+namespace Elastic.Apm.Logging
+{
+	internal static partial class LoggingExtensions
```

Comment:
Could you please explain why this class is partial? 10x :)

Reply:
It shouldn't be! good catch.